processor/src/test/java/com/linecorp/decaton/processor/runtime/ProcessorSubscriptionTest.java

```diff
@@ -170,4 +179,49 @@ public void testOffsetRegression() throws Exception {
         OffsetAndMetadata offset = offsets.get(tp);
         assertEquals(100L, offset.offset());
     }
+
+    @Test(timeout = 10000L)
+    public void testTerminateAsync() throws Exception {
+        TopicPartition tp = new TopicPartition("topic", 0);
+        DecatonMockConsumer consumer = new DecatonMockConsumer();
+        for (int i = 0; i < 10; i++) {
+            consumer.schedulePollTask(
+                    () -> consumer.rebalanceListener.onPartitionsAssigned(consumer.assignment()));
+        }
+        consumer.updateEndOffsets(singletonMap(tp, 10L));
+        byte[] data = {};
```

Comment:
Just `new byte[0]` at creating ConsumerRecord? (creation here with temporary variable made made bit confused)

Reply:
The IDE makes that a warning. Let me try a constant with a (hopefully) clear name.